File: schema-registry-confluent-avro-serializer-consumer/application/src/main/java/sample/consumer/ConfluentAvroConsumerApplication.java
package sample.consumer;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class ConfluentAvroConsumerApplication {
	public static void main(String[] args) {
		SpringApplication.run(ConfluentAvroConsumerApplication.class, args);
	}
}

File: schema-registry-confluent-avro-serializer-consumer/application/src/main/kotlin/sample/consumer/ConfluentAvroConsumerApplication.kt
package sample.consumer

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication


@SpringBootApplication(scanBasePackages = ["com.sample", "sample.consumer"])
class ConfluentAvroConsumerApplication

fun main(args: Array<String>) {
		runApplication<ConfluentAvroConsumerApplication>(*args)
}